server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const port = 3000; 
```

Comment:
Consider using environment variables for your port number. This allows flexibility when deploying to different environments. [important]